resources/views/admin.blade.php

```diff
@@ -117,7 +117,7 @@
                 });
 
                 // Delete Employee
-                $('#myTable tbody').on('click', 'button[id^="btn-dlt"]', function (e) {
+                $('#myTable tbody').on('click', '[id^="btn-dlt"]', function (e) {
```

Comment:
use `id` selector instead of `attribute` selector

Reply:
okay got it.....used class selector and made the change please review @Raza9798  #22